### ISSUE TITLE
Admin Settings My Company - Ensure every form element has a label

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#251]
Title: WCAG 1.3.1,WCAG 3.3.2: Ensures every form element has a label (#o_field_input_594)
Tags: Accessibility, WCAG 1.3.1, WCAG 3.3.2, label

Issue: Ensures every form element has a label (label - https://dequeuniversity.com/rules/axe/3.5/label?application=msftAI)

Target application: My Company - Odoo - http://localhost:8069/web#id=1&active_id=2&model=res.company&view_type=form&cids=1&menu_id=95

Element path: #o_field_input_594

Snippet: `<input class="o_field_char o_field_widget o_input o_address_street" name="street2" placeholder="Street 2..." type="text" id="o_field_input_594">`

<img width="1677" alt="Screen Shot 2020-04-23 at 10 52 20 AM" src="https://user-images.githubusercontent.com/31868510/80113802-c52a8f00-8550-11ea-87c0-8458b6c8750c.png">

How to fix: 
Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  Element has no title attribute or the title attribute is empty

Environment: Chrome version 80.0.3987.163

====

This accessibility issue was found using Accessibility Insights for Web 2.16.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
